scripts/test-matrix.mjs

```diff
@@ -4,124 +4,79 @@
  * Reminder that this script is meant to be runnable without installing
  * dependencies. It can therefore not rely on any external libraries.
  */
-import { spawn, spawnSync } from "node:child_process";
+import { spawn } from "node:child_process";
```

Comment:
I thought we had to use `spawnSync` because of a CVE related to uses of `spawn`?

Reply:
The issue is only when you execute a `.bat` or `.cmd` (or similar) without a shell on Windows. You just need to make sure you set `shell: true` like here: https://github.com/microsoft/react-native-test-app/blob/2163e571573d7ec34efdcca203171ee4a8d4ffa4/test/android/gradle.ts#L131-L133